src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala

```diff
@@ -80,7 +83,7 @@ class AgoraMongoDao(collections: Seq[MongoCollection]) extends AgoraDao {
   }
 
   override def findSingle(entity: AgoraEntity): Option[AgoraEntity] = {
-    val entityVector = find(EntityToMongoDbObject(entity), None)
+    val entityVector = find(entity, None)
```

Comment:
findSingle should be calling the find(entity, projection) method so that it can get the default projections instead of returning the whole object. Alternately we could have it return the fully projected entity, but then we would have two different behaviours for the two find methods.